scripts/clang_tidy_wrapper.sh

```diff
@@ -14,6 +14,9 @@ if grep "$file_relative_to_source" $cmake_source_dir/.clang-tidy-ignore > /dev/n
 	echo "clang-tidy: Ignoring $file_relative_to_source"
 	exit 0
 else
-	exec clang-tidy $file $@
-  exit $?
+	if [ -x /usr/local/bin/brew ]; then
+		exec $(brew --prefix llvm)/bin/clang-tidy $file $@
```

Comment:
Does apple ship it's own clang tidy or what is the reason for this change?

Reply:
Also, why execute v9 when brew is used?

Reply:
> Does apple ship it's own clang tidy or what is the reason for this change?

brew does not link `clang-tidy` by default. As such, it is not guaranteed to exist.

> Also, why execute v9 when brew is used?

Where are you seeing that?

Reply:
Misread that.